components/efuse/esp32c3/include/esp_efuse_table.h

```diff
@@ -156,6 +156,7 @@ extern const esp_efuse_desc_t* ESP_EFUSE_DIS_PAD_JTAG[];
 extern const esp_efuse_desc_t* ESP_EFUSE_DIS_DOWNLOAD_MANUAL_ENCRYPT[];
 extern const esp_efuse_desc_t* ESP_EFUSE_USB_EXCHG_PINS[];
 extern const esp_efuse_desc_t* ESP_EFUSE_VDD_SPI_AS_GPIO[];
+#define ESP_EFUSE_VDD_SPI_AS_GPIO_DESC ESP_EFUSE_VDD_SPI_AS_GPIO
```

Comment:
`// This file was generated from the file esp_efuse_table.csv. DO NOT CHANGE THIS FILE MANUALLY.`

See the comment at the top of this file